doc/develop/west/install.rst

```diff
@@ -53,6 +53,7 @@ West currently supports shell completion in the following shells:
 * bash
 * zsh
 * fish
+* powershell
```

Comment:
As this is currently very limited, we could make that clear? Maybe something like:

```suggestion
* powershell (board qualifiers only)
```